draft-packages/likert-scale-legacy/docs/LikertScaleLegacy.stories.tsx

```diff
@@ -26,70 +29,132 @@ export default {
 }
 
 const scale: Scale = [
+  {
+    value: -1,
+    label: "Not rated",
+  },
   {
     value: 1,
-    label: "Very poor",
+    label: "Strong Disagree",
   },
   {
     value: 2,
-    label: "Poor",
+    label: "Disagree",
   },
   {
     value: 3,
-    label: "Fair",
+    label: "Neither agree or disagree",
   },
   {
     value: 4,
-    label: "Good",
+    label: "Agree",
   },
   {
     value: 5,
-    label: "Very good",
+    label: "Strongly agree",
   },
 ]
 
-const labelId = "456"
-
-export const DefaultStory = () => {
+export const DefaultStory = args => {
   const [selectedItem, setSelectedItem] = useState<ScaleItem | null>(null)
+  const labelId = "456"
 
   return (
-    <div>
+    <>
       <div style={{ marginBottom: "40px" }} id={labelId}>
-        <Heading variant="heading-4">How would you rate this survey?</Heading>
+        <Heading variant="heading-4" color="dark">
+          How would you rate this survey?
+        </Heading>
       </div>
       <LikertScaleLegacy
         scale={scale}
         automationId="123"
         labelId={labelId} // Intended to match the id of the label
         selectedItem={selectedItem}
         onSelect={item => setSelectedItem(item)}
+        {...args}
       />
-    </div>
+    </>
   )
 }
-DefaultStory.storyName = "Default"
+DefaultStory.storyName = "Default (Kaizen Demo)"
```

Comment:
```suggestion
DefaultStory.storyName = "Default (Kaizen Demo)"
DefaultStory.parameters = {
  docs: { source: { type: "code" } },
}
```

The raw code is actually a little clearer than the dynamic one (and is what is currently already out there) :)